airbyte-config/init/src/test/resources/connector_catalog.json

```diff
@@ -12691,7 +12691,7 @@
           "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
@grishick , @evantahler  Do we need to make changes to connector_catalog as well?